fission/main.go

```diff
@@ -323,9 +325,34 @@ func newCliApp() *cli.App {
 
 	app.Before = cliHook
 	app.CommandNotFound = handleCommandNotFound
```

Comment:
Since this line is useless, it's better to remove it in order not to generate confuse.

Reply:
Good point 👍 will fix